immich/_internal/upload.py

```
@@ -0,0 +1,482 @@
+                        )
+                    )
+                if not dry_run:
+                    pbar.update(filepath.stat().st_size)
+            except ApiException as e:
+                msg = str(e)
+                if e.body:
+                    try:
+                        body = json.loads(cast(str, e.body))
+                        msg = str(body.get("message", msg))
+                    except Exception:  # nosec: B110
+                        pass
```

Ruff (0.14.10)

`try`-`except`-`pass` detected, consider logging the exception

(S110)